src/images/next-image-stub.tsx

```diff
@@ -0,0 +1,21 @@
+import * as _NextImage from 'next/image'
+import { ImageProps } from 'next/image'
+
+try {
+  // eslint-disable-next-line @typescript-eslint/no-var-requires
+  const NextImage = require('next/image') as typeof _NextImage
```

Comment:
@RyanClementsHax pretty sure there's a better way here, e.g. you define the code for the stub here, but then in webpack you check which version via package.json the next package that is installed is and load it only when its `> 9`

Reply:
Thanks for pointing this out. I considered this originally but didn't see an obvious way to tackle this via webpack. Do you see a way?